ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/log/FootstepPlannerLogger.java

```diff
@@ -144,7 +165,7 @@ public static void deleteOldLogs(int numberOflogsToKeep, String directory)
       {
          Path earliestLogDirectory = sortedLogFolderPaths.first();
          LogTools.warn("Deleting old log {}", earliestLogDirectory);
-         FileTools.deleteQuietly(earliestLogDirectory);
+         FileUtils.deleteQuietly(earliestLogDirectory.toFile());
```

Comment:
I don't wanna depend on some `ihmc-commons` stuff. We should just depend on a supported library directly. No need to complicate this

Reply:
The code for the FileTools.deleteQuietly is just that
```
public static void deleteQuietly(Path path)
{
   FileUtils.deleteQuietly(path.toFile());
}
```

Reply:
Exactly, whats the benefit of using it if I can just call `FileUtils.deleteQuietly(path.toFile());` directly?